about.html

```diff
@@ -1,261 +1,319 @@
 <!DOCTYPE html>
 <html lang="en">
-	<!-- This file is a placehodler for a page. Rename this file to the name of the page you are working on. -->
+<!-- This file is a placehodler for a page. Rename this file to the name of the page you are working on. -->
 
-	<head>
-		<meta charset="UTF-8" />
-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>If These Lands Could Talk - Page Name</title>
-		<link rel="preconnect" href="https://fonts.googleapis.com" />
-		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-		<link
-			href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
-			rel="stylesheet"
-		/>
-		<link rel="stylesheet" href="/css/normalize.css" />
-		<link rel="stylesheet" href="/css/global.css" />
-		<!-- In a new branch, rename this page.css file to the name of that actual page (Example: home.css) -->
-		<!-- done -->
-		<link rel="stylesheet" href="/css/about.css" />
-	</head>
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>If These Lands Could Talk - Page Name</title>
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap" rel="stylesheet" />
```

Comment:
Looks good, I believe there is another `<link>` with a url for the open sans that needs to be added in for the body text. If you are still waiting for it, please let me know and we can add this in later